opal/mca/memory/patcher/memory_patcher.h

```diff
@@ -0,0 +1,27 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2016      Los Alamos National Security, LLC. All rights
+ *                         reserved.
+ * ******** ADD IBM COPYRIGHT HERE ******
```

Comment:
@gpaulsen @jjhursey @markalle Please fix the IBM copyrights in this code and get @hjelmn to add your commits to this PR.